src/pages/docs/chat/rooms/media-sharing.mdx

```diff
@@ -0,0 +1,165 @@
+---
+title: "Sharing media with Ably Chat"
+meta_description: "How to share media like images, videos, or files with Ably Chat"
+meta_keywords: "image, file, media, sharing, Ably Chat, chat SDK, realtime messaging, dependability, cost optimisation"
+---
+
+A common requirement for chat applications is the ability to share media like images, videos, or files with other participants.
+
+Ably Chat currently does not support media sharing out of the box, but it is very straightforward to link media and metadata to messages, and this can be used to implement media sharing features.
+
+## Architecture Overview
+
+The general approach for implementing media sharing in Ably Chat is:
+
+1. Upload media to your own storage service (your own servers, or a third-party service like AWS S3)
+2. Send a Chat Message with `metadata` containing information about the shared media
+3. On the receiving end, display the media when a message with the relevant metadata is received
+
+## Implementation Guide
+
+We'll build a simple chat room where users can share images with each other.
+
+We assume that `room` is an Ably chat room that is attached. Please refer to our [getting started guide](/docs/chat/getting-started/javascript) for step-by-step instructions on how to set up Ably Chat and get a chat room.
+
+We also assume that a function called `uploadImage() : Promise<string>` exists and handles the whole image upload flow: asks users to pick a picture, uploads it, and returns a unique identifier for this image.
+Let's say that the images will be served from `https://example.com/images/<identifier>`, where the identifier is a string of length between 10 and 15 and matches the `^[a-z0-9]{10,15}$` regex.
+
+### Sending messages with images
```

Comment:
This would become H2 depending on my above comment. But stick to action-orientated headings: `Send a message with an X` 

I say `X` because I think we should be consistent throughout this page in the terminology we use. Is it 'media', 'an attachment', 'a file'.

Reply:
changed them